CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 * Patch-level upgrades to dependencies
     * argonaut-6.2
     * scalaz-7.2.12
+* Allow preambles and epilogues in multipart bodies
+* Limit multipart headers to 10 kilobytes to avoid unbounded buffering
```

Comment:
The two settings will necessarily operate independent of each other (i.e., servlet and AHC people can use this), but there's no reason we can't start from the same default for what's more or less the same problem.